ipie/addons/free_projection/qmc/fp_afqmc.py

```diff
@@ -49,10 +49,19 @@ def __init__(
         mpi_handler,
         params: QMCParamsFP,
         verbose: int = 0,
+        ccsd=None,
     ):
         super().__init__(
-            system, hamiltonian, trial, walkers, propagator, mpi_handler, params, verbose=verbose
+            system,
+            hamiltonian,
+            trial,
+            walkers,
+            propagator,
+            mpi_handler,
+            params,
+            verbose=verbose,
         )
+        self.ccsd = ccsd
```

Comment:
why is this apart of AFQMC now? I think it's just initializing the walker wavefunction right? Just pass in the walkers or add a new factory which expects a CCSD object to initialize the walkers, or build it in the existing builder.